doc/doxygen/riot.doxyfile

```diff
@@ -815,27 +815,44 @@ EXCLUDE_SYMLINKS       = NO
 # exclude all test directories for example use the pattern */test/*
 
 EXCLUDE_PATTERNS       = */board/*/tools/* \
+                         */boards/*/include/arduino_pinmap.h \
+                         */boards/chronos/drivers/include/display.h \
+                         */boards/*/include/periph_conf.h \
+                         */boards/*/include/board.h \
```

Comment:
why would you exclude the board configuration file? I think it makes very much sense to have them in the documentation!